stylelint-config/rules/use-defined-css-var-when-available/props-to-allowed-vars.js

```diff
@@ -48,6 +48,10 @@ const FONT_LINE_HEIGHTS = [
   "--font-line-height-subtext",
 ];
 
+const ANIMATION_TIMING = ["--animation-timing-enter", "--animation-timing-exit", "--animation-timing-transition", "--animation-timing-emphasize", "--expand-animation-timing"];
+
+const ANIMATION_DURATION = ["--animation-productive-short", "--animation-productive-medium", "--animation-productive-medium", "--animation-productive-long", "--animation-expressive-short", "--animation-expressive-medium", "--animation-expressive-long"];
```

Comment:
nitpicking: other arrays of CSS var names are in plural
```suggestion
const ANIMATION_TIMINGS = ["--animation-timing-enter", "--animation-timing-exit", "--animation-timing-transition", "--animation-timing-emphasize", "--expand-animation-timing"];

const ANIMATION_DURATIONS = ["--animation-productive-short", "--animation-productive-medium", "--animation-productive-medium", "--animation-productive-long", "--animation-expressive-short", "--animation-expressive-medium", "--animation-expressive-long"];
```